src/StackExchange.Redis/PhysicalBridge.cs

```diff
@@ -864,20 +979,21 @@ private async Task ProcessBacklogAsync()
                 int failureCount = 0;
 #endif
                 _backlogStatus = BacklogStatus.Starting;
+
                 while (true)
                 {
                     // check whether the backlog is empty *before* even trying to get the lock
-                    if (_backlog.IsEmpty) return; // nothing to do
+                    if (backlog.IsEmpty) return; // nothing to do
 
```

Comment:
If we stick with a 'single backlog' design, can we continue using `_backlog` rather than passing `backlog` into this method?

Reply:
This is a bit of prep for possible vNext - where we feed _another_ bridge's backlog in to process it ;)